### DISCUSSION TITLE
frontend-sample-showcase 3.0

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#3266] [General] [answered]
I am unable to locate code for 3.0 frontend-sample-showcase.   Can you please share location of the repository.

Thanks,

Ovais

**Comment by aruniverse** [comment]
Is there something specific you're looking for? Most of the showcase sample code should be visible to you on the side editor. The only thing not explicitly shown is code for the itwin/sandbox pkg which is mostly just configuration details that shouldn't be pertinent to development outside the sandbox. 

cc @StefanRetief

**Comment by StefanRetief** [comment, marked as answer]
The repository for the frontend sample showcase 3.x is the same repository that was used for the 2.x frontend sample showcase: https://github.com/imodeljs/frontend-sample-showcase. Is this what you're looking for?

**Comment by ovaisa** [comment]
Thank you, yes this is what I was looking for.  

I had a link saved to 2.x imodel mono repo (https://github.com/imodeljs/imodeljs) which contained this repository but now when I use this link, I didn't realize that the browser is automatically routed to a different repo (https://github.com/iTwin/itwinjs-core).  It seems I was looking for the sample showcase in the wrong repo (hence the confusion...)

The sample showcase has been instrumental in our understanding of iTwin.js capabilities.  Looking at the code on the side editor is helpful but following the code execution in VS Code debugger has been invaluable in our understanding of how different capabilities work.  Especially when a sample has multiple source files, following the code in the debugger is only (sane) way to keep track of program execution jumping between different files.

Thanks for pointer to the repo.

**Reply by ovaisa** [reply]
Can you please elaborate what is the difference between the two repositories - https://github.com/imodeljs/ and https://github.com/iTwin

**Reply by calebmshafer** [reply]
Hey @ovaisa, the branding of iModel.js has been superseded by iTwin.js and iTwin. As such we have started to move all active development/repositories over to the [iTwin](https://github.com/iTwin) GitHub Organization. The [imodeljs](https://github.com/imodeljs/) organization is being deprecated and all active development should stop there shortly. The sample showcase is the last repository still managed over in that GitHub Org.
